packages/material-ui/src/Rating/Rating.d.ts

```diff
@@ -133,7 +136,7 @@ export interface RatingProps
    * The size of the component.
    * @default 'medium'
    */
-  size?: 'small' | 'medium' | 'large';
+  size?: OverridableStringUnion<'small' | 'medium' | 'large', RatingPropsSizeOverrides>;
```

Comment:
What is the difference between using and not using `<Record>`?

Reply:
Initially the type expected `Record<string, boolean>`, but recently was updated to this new signature.